project/AI.gd

```diff
@@ -553,6 +584,17 @@ func convert_state_vec_to_interface(state_vec, bad_guys, attack_state):
 	
 	return all_chosen_moves
 
+func move_all_evuls_that_dont_plan(bad_guys, good_guys, flat_board):
```

Comment:
Inte för att jag tror att jag kommer va här o peta, men för läsbarhet kanske den borde heta "get_moves_all_evuls_that_dont_plan()" eller nåt. Man vet först vad den gör när man kommer in i get_moves annars.

Reply:
Sant. Ja tar o döper om den innan jag mergar.